test/python/visualization/test_circuit_latex.py

```diff
@@ -38,6 +39,7 @@ def _get_resource_path(self, filename):
         reference_dir = os.path.dirname(os.path.abspath(__file__))
         return os.path.join(reference_dir, filename)
 
+    @unittest.skipUnless(optionals.HAS_PYLATEX, "needs pylatexenc")
```

Comment:
Is there a reason to not do this skip at the class level it seems like every test method requires pylatexenc to function?

Reply:
No good reason. Fixed in e3f835b8e.